Cargo.toml

```diff
@@ -2,4 +2,11 @@
 
 name = "par-vec"
 version = "0.0.1"
-authors = ["Alexis <a.beingessner@gmail.com>"]
+authors = [
+	"Alexis <a.beingessner@gmail.com>", 
+	"Austin <austin.bonander@gmail.com>"
```

Comment:
Also I wouldn't consider myself an author; this was just generated when I stubbed the file